src/databricks/labs/ucx/queries/assessment/main/07_0_interactive_summary.sql

```diff
@@ -24,10 +24,23 @@ select col1 as pattern, col2 as issue from values
     ('applyInPandas', 			'AF310.1 - applyInPandas'),
     ('mapInPandas', 			'AF310.2 - mapInPandas'),
     ('dbutils.fs.', 			'AF311 - dbutils.fs'),
-    ('dbutils.credentials.', 		'AF312 - credential passthrough') -- credential passthrough
+    ('dbutils.credentials.', 		'AF312 - credential passthrough'), -- credential passthrough,
```

Comment:
Are these AF codes documented anywhere?

Reply:
@FastLee Will be added to docs/assessment.md